src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+import clsx from 'clsx';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  const { content } = selectedTab;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                className={clsx(tab === selectedTab && 'is-active')}
+                data-cy="Tab"
+                key={id}
+              >
+                <a
+                  href={`#${id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    if (tab !== selectedTab) {
+                      onTabSelected(tab);
```

Comment:
To improve readability and maintainability, it is better to move this to a separate handler